class/defaults.yml

```diff
@@ -65,12 +71,19 @@ parameters:
           max:
             cpu: "1"
             memory: "4G"
+            # limits.ephemeral-storage
+            ephemeral-storage: "500Mi"
           min:
             cpu: "10m"
             memory: "4Mi"
+            ephemeral-storage: "100Ki"
```

Comment:
```suggestion
            ephemeral-storage: "5Mi"
```

Reply:
Why set the minimum to such a high value?

Reply:
Well, is it that high? 
Mainly matching the value in `defaultRequest`

Question back: Why only 100Ki?

Reply:
> Question back: Why only 100Ki?

It's off the cuff. A pod not writing to its containers file systems and has no empty dir storage should use almost no ephemeral storage.

The other min values are quite low too...

Reply:
I don't quite understand what you mean, but I have no hard feeling either, so :shrug: